rest-api/model/__init__.py

```diff
@@ -0,0 +1,2 @@
+# This is 64KiB - 1 byte
+MAX_MYSQL_VARCHAR = 65535
```

Comment:
Let's move this to questionnaire_response.py, and make it specific to the field in question.